Python/thread_pthread.h

```diff
@@ -152,6 +152,24 @@ PyThread__init_thread(void)
  * Thread support.
  */
 
+typedef struct {
```

Comment:
Please add a comment explaining this is necessary for adapting the function type.

Reply:
Done.

Reply:
Oops, I missed this one. I'll add the comment.

Reply:
Actually done now.